src/components/aside.js

```diff
@@ -0,0 +1,50 @@
+import { SearchForm, Card } from './common-small';
+import ProductsMenu from './products-menu';
+import TypeCheckList from './type-check-list';
+
+function TypeCheckListType() {
+  return (
+    <TypeCheckList>
+      <h6 className='type-check-list__title'>Type</h6>
+      <SearchForm />
+    </TypeCheckList>
+  );
+}
+
+function TypeCheckListBrand() {
+  return (
+    <TypeCheckList>
+      <h6 className='type-check-list__title'>Type</h6>
+    </TypeCheckList>
+  );
+}
+function RatingsList() {
+  return (
+    <TypeCheckList>
+      <h6 className='type-check-list__title'>Ratings</h6>
+    </TypeCheckList>
+  );
+}
+
+function OthersCategories() {
+  return (
+    <Card title='Refine by'>
+      <TypeCheckListBrand classes='list' />
+      <TypeCheckListType classes='list' />
+      <RatingsList classes='list' />
+    </Card>
+  );
+}
+
+function filterPanel() {

```

Comment:
component đặt tên in hoa chứ Hải ơi. Tên file cũng là `Aside.js` hoặc `Aside.jsx` chứ
```suggestion
function FilterPanel() {
```